tests/integration/test_ssm.py

```diff
@@ -33,22 +37,25 @@ def test_put_parameters(self, ssm_client, create_parameter):
         )
 
         _assert(param_name, param_name, ssm_client)
-        _assert(f"/{param_name}", param_name, ssm_client)  # TODO: not valid
+        _assert(f"/{param_name}", f"/{param_name}", ssm_client)
 
-    # TODO botocore.exceptions.ClientError: An error occurred (ValidationException) when calling the GetParameter operation: Parameter name: can't be prefixed with "ssm" (case-insensitive). If formed as a path, it can consist of sub-paths divided by slash symbol; each sub-path can be formed as a mix of letters, numbers and the following 3 symbols .-_
+    @pytest.mark.aws_validated
     def test_hierarchical_parameter(self, ssm_client, create_parameter):
-        param_a = f"{short_uid()}"
+        param_a = short_uid()
         create_parameter(
             Name=f"/{param_a}/b/c",
             Value="123",
             Type="String",
         )
 
         _assert(f"/{param_a}/b/c", f"/{param_a}/b/c", ssm_client)
-        _assert(f"/{param_a}//b//c", f"/{param_a}/b/c", ssm_client)
-        _assert(f"{param_a}/b//c", f"/{param_a}/b/c", ssm_client)
+        for pname in [f"/{param_a}//b//c", f"{param_a}/b/c"]:
+            with pytest.raises(Exception) as exc:
```

Comment:
nit: an alternative could be to parametrize the test, which makes debugging the test much easier if it fails